pro/src/pages/Offerers/Offerer/VenueV1/fields/PricingPoint/PricingPoint.tsx

```diff
@@ -99,13 +106,16 @@ export const PricingPoint = ({ offerer, venue }: PricingPointProps) => {
         onConfirm={handleClick}
         icon={strokeValidIcon}
         title="Êtes-vous sûr de vouloir sélectionner"
-        secondTitle={`ce lieu avec SIRET\u00a0?`}
+        secondTitle={`${isOfferAddressEnabled ? 'cette structure' : 'ce lieu'} avec SIRET\u00a0?`}
         isLoading={isSubmitingPricingPoint}
         open={isConfirmSiretDialogOpen}
       >
         <p className={styles['text-dialog']}>
-          Vous avez sélectionné un lieu avec SIRET qui sera utilisé pour le
-          calcul de vos remboursements
+          Vous avez sélectionné{' '}
+          {isOfferAddressEnabled
+            ? 'une structure avec SIRET qui sera utilisée'
+            : 'un lieu avec SIRET qui sera utilisé'}{' '}
+          avec SIRET qui sera utilisé pour le calcul de vos remboursements
```

Comment:
```suggestion
          pour le calcul de vos remboursements
```
Ça fait une répétition de "avec SIRET qui sera utilisé" =)